javascript/javascript1/week2/weatherWear.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Hack Your Future, Javascript, homework Week2
+ * this file includes:
+   Weather wear program that shows what the user should wear based on the temperature.
+*/
+
+function userWear(temperature) {
+    if (temperature >= 25) {
+        return 'shorts and a t-shirt';
+    } else if (temperature >= 15 && temperature < 25) {
```

Comment:
Because you can get here unless temperature is smaller then 25
```suggestion
    } else if (temperature >= 15) {
```

Reply:
Thanks. its done.